twake/frontend/src/app/scenes/Client/WorkspacesBar/WorkspacesBar.js

```diff
@@ -40,7 +41,7 @@ export default class WorkspacesBar extends Component {
     }
 
     return (
-      <div className="workspaces_view fade_in">
+      <Layout.Sider width="60px" className="workspaces_view fade_in">
```

Comment:
Can we put the 60px value in the css class workspaces_view ?

Reply:
We need to keep the width prop or antd design will add a default style :
![Capture d’écran de 2020-10-22 17-59-12](https://user-images.githubusercontent.com/36481167/96900750-bb97ca80-1492-11eb-8a80-065d0a7ea39e.png)